src/action_runner/action_runner.vcxproj

```diff
@@ -22,7 +23,7 @@
     <VCProjectVersion>16.0</VCProjectVersion>
     <ProjectGuid>{D29DDD63-E2CF-4657-9FD5-2AEDE4257E5D}</ProjectGuid>
     <RootNamespace>actionrunner</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
```

Comment:
What is the reason to set `10.0.17134.0` in stead of `10.0.18362.0`?

Reply:
Aligns with the README: `Microsoft PowerToys requires Windows 10 1803 (build 17134) or later.` But we can change to whatever you want the min target to be.

Reply:
Since we have been testing and building the release with 10.0.18362.0 let's set it as the default.

Reply:
I'm okay with bumping up to `10.0.17763.0` (EOL November 10, 2020), to re-align with the Windows 10 support lifecycle, but anything beyond will cause misalignment with the minimum OS requirement listed in the README.

I'm okay with bumping this to `10.0.17763.0`, but this will mean:
* Minimum supported OS will need to change in README
* This will signal that PowerToys does not support Windows 10 Education or Enterprise lifecycles (17134 is still supported until late Q4)

Official builds should not be using the `10.0.18362.0` SDK.

I'm not familiar with the team hierarchy here yet (apologies) so tagging @crutkas for double-check.

Reply:
Let me re-check what are we using in the piepline to build the release.

Reply:
Our build environment is 1809 with VS 16.4.3.
@crutkas can you confirm is it OK to set it as minimum version supported?

Reply:
I'm fine moving to RS5 as min version looking at telem.

Reply:
So, to confirm, you do not want to support Windows 10 Education or Enterprise, right? So PowerToys moving forward will only support consumer lifecycle.

Reply:
I misunderstood, yes, we should support those.  I thought RS5 flip was May timeframe ,not Nov.  We currently support Rs4+ so lets keep that until nov.